cmd/oc-mirror/main.go

```diff
@@ -1,24 +1,19 @@
 package main
 
 import (
-	"fmt"
-	"github.com/containers/buildah"
-	"github.com/containers/storage/pkg/unshare"
 	"github.com/openshift/oc-mirror/pkg/cli/mirror"
 	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
-	"os"
 )
 
 func main() {
-	if testE2E := os.Getenv("TEST_E2E"); len(testE2E) != 0 {
-		fmt.Println("envar TEST_E2E detected - bypassing unshare")
-	} else {
-		if buildah.InitReexec() {
-			return
-		}
-		unshare.MaybeReexecUsingUserNamespace(false)
-	}
-
+	// if testE2E := os.Getenv("TEST_E2E"); len(testE2E) != 0 {
```

Comment:
@sherine-k -  should we remove this completely and add it in when we look at buildah again ?